pdns/dnsdist.cc

```diff
@@ -2661,7 +2901,23 @@ int main(int argc, char** argv)
     }
     handleQueuedHealthChecks(*mplexer, true);
 
+    for (auto i : g_xsk) {
```

Comment:
Likely needs to be guarded by `#ifdef HAVE_XSK` as well?

Reply:
Done.